guides/front_end/react_quick_start.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: React quick start — Meilisearch documentation
+description: Learn how to integrate a search-as-you-type experience into your React app.
+---
+
+# React quick start
+
+Learn how to integrate a search-as-you-type experience into your React app.
+
+## 1. Create a React application
+
+Create your React application using a [Vite](https://vitejs.dev/) template:
+
+```bash
+npm create vite@latest my-app -- --template react
+```
+
+## 2. Install the library of search components
+
+The fastest way to get started is to use:
+- [React InstantSearch](https://github.com/algolia/instantsearch/): an open-source library that provides all the front-end tools you need to highly customize your search bar environment
+- [instant-meilisearch](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/instant-meilisearch): the Meilisearch client to establish the communication between your Meilisearch instance and the React InstantSearch library
+- [instantsearch.css](https://github.com/algolia/instantsearch/tree/master/packages/instantsearch.css) (optional) is the CSS library to style the search components. Although optional, using it adds basic styles that can significantly enhance your search UI
```

Comment:
I'd recommend moving this beneath the code sample. This way people who just want to get stuff done quickly don't need to read through all that text.

I'd also shorten it by not describing each package individually.